test/e2e/tests/state-snapshots/sentry-mask-policy-gaps.json

```diff
@@ -0,0 +1,130 @@
+  "announcements.4.id",
+  "announcements.4.image.src",
+  "announcements.4.image.width",
+  "announcements.4.isShown",
+  "announcements.5.date",
+  "announcements.5.id",
+  "announcements.5.isShown",
+  "announcements.6.date",
+  "announcements.6.id",
+  "announcements.6.isShown",
+  "announcements.7.date",
+  "announcements.7.id",
+  "announcements.7.isShown",
+  "announcements.8.date",
+  "announcements.8.id",
+  "announcements.8.isShown",
+  "announcements.9.date",
+  "announcements.9.id",
+  "announcements.9.image.src",
+  "announcements.9.image.width",
+  "announcements.9.isShown",
+  "announcements.10.date",
+  "announcements.10.id",
+  "announcements.10.image.src",
+  "announcements.10.image.width",
+  "announcements.10.isShown",
+  "announcements.11.date",
+  "announcements.11.id",
+  "announcements.11.isShown",
+  "announcements.12.date",
+  "announcements.12.id",
+  "announcements.12.image.src",
+  "announcements.12.image.width",
+  "announcements.12.isShown",
+  "announcements.13.date",
+  "announcements.13.id",
+  "announcements.13.isShown",
+  "announcements.14.date",
+  "announcements.14.id",
+  "announcements.14.isShown",
+  "announcements.15.date",
+  "announcements.15.id",
+  "announcements.15.isShown",
+  "announcements.16.date",
+  "announcements.16.id",
+  "announcements.16.isShown",
+  "announcements.17.date",
+  "announcements.17.id",
+  "announcements.17.isShown",
+  "announcements.18.date",
+  "announcements.18.id",
+  "announcements.18.isShown",
+  "announcements.19.date",
+  "announcements.19.id",
+  "announcements.19.isShown",
+  "announcements.20.date",
+  "announcements.20.id",
+  "announcements.20.isShown",
+  "announcements.21.date",
+  "announcements.21.id",
+  "announcements.21.isShown",
+  "announcements.22.date",
+  "announcements.22.id",
+  "announcements.22.isShown",
+  "announcements.23.date",
+  "announcements.23.id",
+  "announcements.23.isShown",
+  "browserEnvironment.browser",
+  "browserEnvironment.os",
+  "customNonceValue",
+  "database",
+  "disabledRpcMethodPreferences.eth_sign",
+  "identities.0x5cfe73b6021e818b776b421b1c4db2474086a7e1.address",
+  "identities.0x5cfe73b6021e818b776b421b1c4db2474086a7e1.lastSelected",
+  "identities.0x5cfe73b6021e818b776b421b1c4db2474086a7e1.name",
+  "isAccountMenuOpen",
+  "isInitialized",
+  "isNetworkMenuOpen",
+  "keyringTypes.0",
+  "keyringTypes.1",
+  "keyringTypes.2",
+  "keyringTypes.3",
+  "keyringTypes.4",
+  "keyringTypes.5",
+  "lastUpdated",
+  "networkConfigurations.networkConfigurationId.chainId",
+  "networkConfigurations.networkConfigurationId.id",
+  "networkConfigurations.networkConfigurationId.networkConfigurationId",
+  "networkConfigurations.networkConfigurationId.nickname",
+  "networkConfigurations.networkConfigurationId.rpcUrl",
+  "networkConfigurations.networkConfigurationId.ticker",
+  "networksMetadata.networkConfigurationId.EIPS.1559",
+  "networksMetadata.networkConfigurationId.status",
+  "nextNonce",
+  "pendingApprovalCount",
```

Comment:
This would be nice to include (it's in the ApprovalController)

Reply:
That's the correct place, good catch. I'm not sure why that's marked as `anonymous: false`. Sometimes that setting is overlooked or misunderstood - it's easy for that to happen because it's not used for anything functional yet.

Leaving it out to stay on the safe side would be reasonable though.